backend/substrapp/compute_tasks/command.py

```diff
@@ -97,13 +100,26 @@ def _get_args(ctx: Context) -> list[str]:  # noqa: C901
 
     rank = str(task.rank)
     if rank and task_category != orchestrator.ComputeTaskCategory.TASK_PREDICT:
-        command += ["--rank", rank]
+        args += ["--rank", rank]
     if ctx.has_chainkeys:
         inputs.append(TaskResource(id=TASK_IO_CHAINKEYS, value=chainkeys_folder))
 
-    command += ["--inputs", f"'{json.dumps(inputs)}'"]
-    command += ["--outputs", f"'{json.dumps(outputs)}'"]
+    args += ["--inputs", json.dumps(inputs)]
+    args += ["--outputs", json.dumps(outputs)]
 
-    logger.debug("Generated task command", command=command)
+    logger.debug("Generated substra-tools arguments", args=args)
 
-    return command
+    return args
+
+
+def write_command_args_file(dirs: Directories, command_args: list[str]) -> None:
+    """Write the substra-tools command line arguments to a file.
+
+    The format uses one line per argument. See
+    https://docs.python.org/3/library/argparse.html#fromfile-prefix-chars
+    """
+    path = os.path.join(dirs.task_dir, TaskDirName.CliArgs, Filenames.CliArgs)
+
+    with open(path, "w") as f:
```

Comment:
Out of scope for this PR, but it would be nice to be able to write:
```suggestion
    with open(dirs.arguments_file, "w") as f:
```